tarantool/request.py

```diff
@@ -84,8 +85,13 @@ def __init__(self, conn):
         # The option controls whether to pack binary (non-unicode)
         # string values as mp_bin or as mp_str.
         #
-        # The default behaviour of the connector is to pack both
-        # bytes and Unicode strings as mp_str.
+        # The default behaviour of the Python 2 connector is to pack
+        # both bytes and Unicode strings as mp_str.
+        #
+        # The default behaviour of the Python 3 connector (since
+        # default encoding is "utf-8") is to pack bytes as mp_bin
+        # and Unicode strings as mp_str. encoding=None mode must
+        # be used to work with non-utf strings.
```

Comment:
I plan to work on https://github.com/tarantool/tarantool-python/issues/67 issue after merging #207, #211, #213 since many things in doc and README are outdated and need to be fixed. I think it is better to do detailed description as part of this PR since there is no up-to-date section where I can add this info to.